modules/app_books.rb

```diff
@@ -0,0 +1,18 @@
+require './book'
+
+module Books
+  def list_all_books(books)
+    puts 'No book added yet' if books.empty?
+    books.map { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+  end
+
+  def create_book(books)
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+    book = Book.new(title, author)
+    books << book
+    puts 'Book created successfully'
+  end
```

Comment:
- [ ] in this method you are printing to the console, getting data from the user and saving the data as well, this is not in line with the single responsibility rules, please have another look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more details.